perf.groovy

```diff
@@ -0,0 +1,32 @@
+// Import the utility functionality.
+
+import jobs.generation.*;
+
+def project = GithubProject
+def branch = GithubBranchName
+def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolderName(branch)
+
+
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        def newJob = job(Utilities.getFullJobName(project, "perf_${os}", isPR)) {
+            // Set the label.
+            steps {
+                    // Batch
+                    batchFile("set __TestIntermediateDir=int&&build.cmd release x64")
+                    batchFile("tests\\runtest.cmd release x64")
```

Comment:
Runtest failed after adding "GenerateLayoutOnly" during cleanup, and didn't have a neat solution. Failure logs are pasted below, which is from the full log: http://dotnet-ci2.cloudapp.net/job/smile21prc_coreclr/job/coreclr-perf/job/perf_Windows_NT/9/consoleFull.

For now, I'll hold adding this switch, and minimize test run time later when it's clean.

12:46:55 RUNTEST: Invoking msbuild
12:46:58 D:\j\workspace\perf_Windows_NTf52db88c\tests\publishdependency.targets(32,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64": { } }' to your project.json and then re-run NuGet restore. [D:\j\workspace\perf_Windows_NTf52db88c\tests\runtest.proj]
12:46:58 D:\j\workspace\perf_Windows_NTf52db88c\tests\publishdependency.targets(32,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64": { } }' to your project.json and then re-run NuGet restore. [D:\j\workspace\perf_Windows_NTf52db88c\tests\runtest.proj]
12:47:09 
12:47:09 D:\j\workspace\perf_Windows_NTf52db88c\tests\publishdependency.targets(32,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64": { } }' to your project.json and then re-run NuGet restore. [D:\j\workspace\perf_Windows_NTf52db88c\tests\runtest.proj]
12:47:09 D:\j\workspace\perf_Windows_NTf52db88c\tests\publishdependency.targets(32,5): warning : Your project.json doesn't have a runtimes section. You should add '"runtimes": { "win7-x64": { } }' to your project.json and then re-run NuGet restore. [D:\j\workspace\perf_Windows_NTf52db88c\tests\runtest.proj]
12:47:09 RUNTEST:  Created the runtime layout with all dependencies in D:\j\workspace\perf_Windows_NTf52db88c\tests\..\bin\tests\Windows_NT.x64.Release\Tests\Core_Root
12:47:09 Build step 'Execute Windows batch command' marked build as failure
12:47:10 [BFA] Scanning build for known causes...
12:47:11 ......[BFA] No failure causes found
12:47:17 [BFA] Done. 6s
12:47:17 Notifying endpoint 'HTTP:https://jaredpar.azurewebsites.net/api/BuildEvent?code=tts2pvyelahoiliwu7lo6flxr8ps9kaip4hyr4m0ofa3o3l3di77tzcdpk22kf9gex5m6cbrcnmi'
12:47:17 [WS-CLEANUP] Deleting project workspace...Cannot delete workspace :remote file operation failed: D:\j\workspace\perf_Windows_NTf52db88c at hudson.remoting.Channel@f7d5681:Azure0810113103: java.io.IOException: Unable to delete 'D:\j\workspace\perf_Windows_NTf52db88c\Tools\net45\Microsoft.DotNet.Build.Tasks.dll'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
12:47:35 ERROR: Step ‘Delete workspace when build is done’ failed: Cannot delete workspace: remote file operation failed: D:\j\workspace\perf_Windows_NTf52db88c at hudson.remoting.Channel@f7d5681:Azure0810113103: java.io.IOException: Unable to delete 'D:\j\workspace\perf_Windows_NTf52db88c\Tools\net45\Microsoft.DotNet.Build.Tasks.dll'. Tried 3 times (of a maximum of 3) waiting 0.1 sec between attempts.
12:47:35 Finished: FAILURE